docs/guides/admin/docs/releasenotes.md

```diff
@@ -51,6 +51,7 @@ Configuration changes
     - `etc/org.opencastproject.security.urlsigning.provider.impl.GenericUrlSigningProvider.cfg`
     - `etc/org.opencastproject.security.urlsigning.provider.impl.WowzaUrlSigningProvider.cfg`
     - `etc/org.opencastproject.security.urlsigning.verifier.impl.UrlSigningVerifierImpl.cfg`
+- In the `org.opencastproject.organization-mh_default_org.cfg` file change the variable of the events ID in the player properties (prop.player) to avoid issues with Ansible or other type of deployment software.
```

Comment:
Maybe something like:

Due to [MH-13569](https://opencast.jira.com/browse/MH-13569), the value of the property prop.player in`org.opencastproject.organization-mh_default_org.cfg`needs to be adjusted 

Idea: You don't need to explain here why this has been done or how exactly it is done. Rather just indicate that the value prop.player has been adjusted.